flink-core/src/main/java/org/apache/flink/configuration/WebMonitorOptions.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.configuration;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/**
+ * Configuration options for the WebRuntimeMonitor
+ */
+@PublicEvolving
+public class WebMonitorOptions {
+
+	/**
+	 * Timeout for asynchronous operations by the WebRuntimeMonitor
+	 */
+	public static final ConfigOption<Long> WEB_TIMEOUT = ConfigOptions
+		.key("web.timeout")
```

Comment:
Yes, see my comments above. I think the existing web monitor options are a bit misleading because the web monitor is not necessarily bound to the `JobManager`.